elastic/apm-server
# Send APM Server config to Kibana

## Motivation/summary

Send a sanitized and flattened version of the apm-server config to kibana on
startup.

Closes https://github.com/elastic/apm-server/issues/5377

## Checklist




For functional changes, consider:
- Is it observable through the addition of either **logging** or **metrics**?
- Is its use being published in **telemetry** to enable product improvement?
- Have system tests been added to avoid regression?

## How to test these changes

Start up kibana, elasticsearch and apm-server. Verify that the apm-server
config is indexed in elasticsearch.

## Related issues

Depends on https://github.com/elastic/kibana/issues/100657
